sdk/schemaregistry/azure-schemaregistry/README.md

```diff
@@ -85,7 +85,7 @@ from azure.identity import DefaultAzureCredential
 from azure.schemaregistry import SchemaRegistryClient
 
 token_credential = DefaultAzureCredential()
-fully_qualified_namespace = os.environ['SCHEMA_REGISTRY_FULLY_QUALIFIED_NAMESPACE']
+fully_qualified_namespace = os.environ['SCHEMAREGISTRY_AVRO_FULLY_QUALIFIED_NAMESPACE']
```

Comment:
is this change necessary? is there automation that actually runs these code snippets?

Reply:
nothing runs the snippets in the README, but I wanted these to work when copy pasted + to reflect what we have in the samples.